config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -52,10 +52,17 @@ presubmits:
     always_run: false
     optional: true
     decorate: true
+    extra_refs:
+    - base_ref: master
+      path_alias: k8s.io/kubernetes
```

We don't need these, instead:
```suggestion
    path_alias: k8s.io/kubernetes
```

---

Changed here :)